docs/buildah-manifest-inspect.1.md

```diff
@@ -16,6 +16,17 @@ Displays the manifest list or image index stored using the specified image name.
 
 A formatted JSON representation of the manifest list or image index.
 
+## OPTIONS
+
+**--authfile** *path*
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+
+**--tls-verify** *bool-value*
+
+Require HTTPS and verification of certificates when talking to container registries (defaults to true).  TLS verification cannot be used when talking to an insecure registry.
```

Comment:
Nit: the new flags are not tested in the commit adding them but the one after.

Reply:
I was not planning to add these flags but they were needed for the `--add-compression` test, I can add independent tests for these but they will be just clone of `--add-compression` test without the `--add-compression` flag, I decided not to add because of duplication since these two flags are already getting tested in the test added for `--add-compression`. But I could be wrong :sweat_smile: